mindjunkies/forums/tests/test_views.py

```
@@ -222,8 +215,10 @@ def test_post_valid_comment(self):
         setattr(request, "_messages", messages)
 
         view = CommentSubmissionView.as_view()
-        view(
-            request, course_slug=self.course.slug, topic_slug=self.topic.slug
+        response = view(
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)